myHashMap.py

```diff
@@ -0,0 +1,84 @@
+#Time Complexity : O(n)- Here n is 1000, as I have defined size as 1000
```

Comment:
This is wrong way of thinking about Time Complexity

Reply:
The TC is O(n), where n will be the number of elements present in stack and not the size defined in the code. Is that right?